### PR TITLE
BZ:2040457 - Updated known issue in RN with additional information.

### PR DESCRIPTION
**Pull request by MirzWeiss**
Bugzilla [2040457](https://bugzilla.redhat.com/show_bug.cgi?id=2040457) update to Sandboxed containers release notes in 4.9 and main.

Preview link: https://deploy-preview-41252--osdocs.netlify.app/openshift-enterprise/latest/sandboxed_containers/sandboxed-containers-4.9-release-notes.html#sandboxed-containers-1-1-known-issues

Reviewed and approved by SME and QE.

Went through change management.

### CODE REVIEW

File: sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc [hunk 1]
```diff
@@ -47,7 +47,13 @@ You can now install the {sandboxed-containers-operator} in a disconnected enviro
 [id="sandboxed-containers-1-1-known-issues"]
 == Known issues
 
-* If you are using {sandboxed-containers-first}, you cannot use the `hostPath` volume in an {product-title} cluster to mount a file or directory from the host node’s file system into your pod. As an alternative, you can use local persistent volumes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1904609[*BZ#1904609*])
+* If you are using {sandboxed-containers-first}, you may receive SELinux denials accessing files or directories mounted from the `hostPath` volume in an {product-title} cluster. These denials can occur even when running privileged sandboxed containers, since privileged sandboxed containers do not disable SELinux checks.
```
Comment (bmcelvee) [inline sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc:50]: s/may/might

File: sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc [hunk 2]
```diff
@@ -47,7 +47,13 @@ You can now install the {sandboxed-containers-operator} in a disconnected enviro
...
++
+Following SELinux policy on the host guarantees full isolation of the host file system from the sandboxed workload by default, and provides stronger protection against potential security flaws in `virtiofsd` or QEMU.
++
+If the mounted files or directories do not have specific SELinux requirements on the host, you can use local persistent volumes as an alternative. Files are automatically relabeled to container_file_t, following SELinux policy for container runtimes. See xref:../storage/persistent_storage/persistent-storage-local.adoc[Persistent storage using local volumes] for more information.
```
Comment (bmcelvee) [inline sandboxed_containers/sandboxed-containers-4.9-release-notes.adoc:54]: s/container_file_t/`container_file_t` 

(Add backticks to container_file_t)

**Comment by bmcelvee** [comment]
Just a couple of suggestions. Otherwise, LGTM! 

Does this one need to go through the change management process? It looks like it might since it's updating a known issue.

**Comment by MirzWeiss** [comment]
> Does this one need to go through the change management process? It looks like it might since it's updating a known issue.
 
It does! Making the changes and then sending it out :)

**Comment by cpmeadors** [comment]
lgtm

**Pull request merged**